app/components/repeated-save-form.ts

```diff
@@ -218,14 +241,35 @@ export default class RepeatedSaveFormComponent extends Component {
       );
     }
 
+    // Evasion can only be applied to saves which deal half damage on
+    // a passed save. (It also can only be applied to dexterity saves,
+    // but this is beyond our power to enforce.)
+    if (this.targetHasEvasion && !this.halfDamageOnPass) {
+      // TODO: Surface this error more visibly
```

Comment:
Since the evasion checkbox is disabled and un-checked by switching spells to deal no damage on a passed save, this fix is not urgent--it would be quite difficult to get the site into this state.